src/main/java/com/minecolonies/coremod/colony/workorders/AbstractWorkOrder.java

```diff
@@ -408,7 +408,7 @@ public void serializeViewNetworkData(@NotNull final PacketBuffer buf)
         buf.writeInt(getType().ordinal());
         buf.writeString(get());
         buf.writeBlockPos(buildingLocation == null ? BlockPos.ZERO : buildingLocation);
-
+        buf.writeInt(0);
```

Comment:
what is this about?

Reply:
It's the level, if it's not a building, the level is 0 (every builder can build any scan)